src/components/sharing/ShareBox/index.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+var _           = require('lodash');
+var React       = require('react');
+var I18nMixin   = require('../../mixins/I18n');
+var Input       = require('../../forms/Input');
+var ShareIcon   = require('../ShareIcon');
+
+module.exports = React.createClass({
+  displayName: "ShareBox",
+
+  toString: function(obj) {
+    return _.map(obj, function(value, key) {
+      return encodeURIComponent(key) + '=' + encodeURIComponent(value);
```

Comment:
Not really sure if you need to encodeURIComponent here. The values passed to the window.open specs param are plain text and simple values.